backend/typescript/types.ts

```diff
@@ -38,8 +38,8 @@ export type CamperDTO = {
   allergies: string;
   hasCamera: boolean;
   hasLaptop: boolean;
-  earlyDropoff: string;
-  latePickup: string;
+  earlyDropoff: Date[];
```

Comment:
I don't believe Date objects are supported for DTOs. We should use strings and rely on validators to ensure they are in the proper format

Reply:
registration date is also a Date format, so I thought it was ok. Lmk!

Reply:
Hmm ok, if it works in testing maybe it's fine because of type conversions